### ISSUE TITLE
Add CI performance and scaling details

### ISSUE THREAD

**Issue by bspitzbart** [iceberg-project/iceberg-project.github.io#9]
Can we make the Jupyter notebook performance and scaling test results concise and understandable to a wide audience?

**Comment by iparask** [comment]
We can! The question is how should we publish something like that and whether or we are destroying the data that we get for future publications?

**Comment by AymenFJA** [comment]
yes, we can, it is a good idea to have a Jupyter notebook, that gives a general preview of what are the results and what we achieved now.
